composer.json

```diff
@@ -27,7 +27,12 @@
     },
 
     "require-dev": {
-        "mink/driver-testsuite": "dev-master"
+        "mink/driver-testsuite": "dev-master",
+        "symfony/phpunit-bridge": "4.0.*"
```

Comment:
Please do not change `composer.json` at all to make Travis CI happy.

Right now because of current change you're blocking anyone, who is using this package in their choice of PHPUnit version on assumption, that they're be using `driver-testsuite` test case class, that is used only to test Mink drivers itself, but not the apps using Mink or it's drivers.

Reply:
The reason why I added this is that phpunit-bridge > 4.1 added a short array syntax here:

https://github.com/symfony/phpunit-bridge/blob/4.1/DeprecationErrorHandler.php#L95

but the minimum requirement for that version is PHP >= 5.3.3:

https://packagist.org/packages/symfony/phpunit-bridge#v4.1.0

I found a different solution though 😄 .